SparkPost/matchbox
# UX-154 Use Peer Dependencies

### What Changed
- `matchbox` and `matchbox-icons` now specify `react` `react-dom` and `styled-components` as peer dependencies and dev dependencies 

### How To Test or Verify
- Ensure your environment is up to date, `npm run clean` and `rm -rf node_modules` in the root directory and package directories.
- Reinstall deps with `npm i` `lerna bootstrap`
- Verify storybook and package builds work

### PR Checklist
